mrbgems/mruby-os-memsize/test/memsize.rb

```diff
@@ -26,14 +26,20 @@
     @a = 1
     @b = 2
   end
-  class_total_size = empty_class_def_size + (int_size * 2)
+  ivnum = 2
+  segnum = ((ivnum + Mrbtest::COUNTOF_IV_SEGMENT - 1) / Mrbtest::COUNTOF_IV_SEGMENT).floor
+  class_total_size = empty_class_def_size + (int_size * ivnum) +
+                     Mrbtest::SIZEOF_IV_SEGMENT * segnum
```

Comment:
I know this is kind of controversial, but I don't think we need these precise tests for `memsize_of`.
The fundamental behavior of `memsize_of` is "you can get the size of a various object in integer".

Reply:
This test itself has been deleted.